AppCenter/AppCenter/Internals/Channel/MSChannelUnitDefault.m

```diff
@@ -100,20 +106,22 @@ - (void)enqueueItem:(id<MSLog>)item {
     MSLogWarning([MSAppCenter logTag], @"Log is not valid.");
     return;
   }
+  
+  // Additional preparations for the log. Used to specify the session id and distribution group id.
```

Comment:
The session id and distribution group id should be set **before** saving the log to the buffer

Reply:
That's right